demo05_basic_defender.py

```diff
@@ -10,14 +10,13 @@
 
 import networkx
 
-from utils import walls_to_graph
+from pelita.utils import walls_to_graph
 
 
 def move(bot, state):
-
-    if state is None:
+    if 'graph' not in state:
```

Comment:
This should be consistent with all the other demos, so it must read `if state == {}` or, if my proposal gets through, `if bot.turn==0 and bot.round==1`.